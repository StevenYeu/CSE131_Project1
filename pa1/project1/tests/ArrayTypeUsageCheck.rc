function : void main(){
//    int x = 5;
//    x[2] = 1;
//    int a[2];
//    a[1.1] = 0;
    const int cc = 1;
    float a[10];
    float b[10];
    float c[2][10];
    float *d;
    c[1][9] = a[9];
    b[b[b[b[2] -cc] + cc] -1] = c[1][9];
    a[2] = d[0] + d[99999];
}
