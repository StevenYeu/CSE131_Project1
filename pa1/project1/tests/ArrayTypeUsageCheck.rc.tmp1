  Type of index expression in array reference (float) not equivalent to int.
Compile: failure.
