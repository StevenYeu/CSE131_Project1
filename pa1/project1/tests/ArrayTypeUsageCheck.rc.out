Error, "/tmp/cs131_testrunner_oNarMF/file.rc": 
  Type of index expression in array reference (float) not equivalent to int.
Compile: failure.
